src/Pos/CompileConfig/Type.hs

```diff
@@ -16,7 +16,7 @@ data CompileConfig = CompileConfig
     {
       ccNetworkDiameter               :: !Int
       -- ^ Estimated time for broadcasting messages
-    , ccDefaultPeers                  :: ![String]
+    , ccDefaultPeers                  :: ![Text]
```

Comment:
Not really, I deleted all usages of `Text.Parsec.String` in favor of `Text.Parsec.Text` as it's written in one of commit messages.
Previously there was a mix and it was annoying and at some point I had to use the same parser for `String` and `Text` and didn't want to copy-paste it.